core/src/test/java/org/infinispan/expiration/impl/ClusterExpirationFunctionalTest.java

```diff
@@ -27,7 +36,8 @@
  * @since 8.0
  */
 @Test(groups = "functional", testName = "expiration.impl.ClusterExpirationFunctionalTest")
-@InCacheMode({CacheMode.DIST_SYNC, CacheMode.REPL_SYNC, CacheMode.SCATTERED_SYNC})
+@InCacheMode({CacheMode.DIST_SYNC, CacheMode.REPL_SYNC})
+@InTransactionMode({TransactionMode.NON_TRANSACTIONAL, TransactionMode.TRANSACTIONAL})
```

Comment:
I'd rather write a factory() method than create a subclass for scattered cache.

Reply:
Okay